QgisModelBaker/gui/panel/pg_config_panel.py

```diff
@@ -20,7 +20,7 @@
 from enum import IntEnum
 
 try:
-    import QgisModelBaker.libs.pgserviceparser
+    import QgisModelBaker.libs.pgserviceparser as pgserviceparser
```

Comment:
@signedav not sure why the tests were failing locally without this... does it make sense for you?

Reply:
locally are not using this import right?

Reply:
Thanks for the explanation :)

Reply:
This should work as well

```
from QgisModelBaker.libs import pgserviceparser
```